.env

```diff
@@ -8,12 +8,14 @@ DATABASE_PORT=5432
 DATABASE_DB=airbyte
 # translate manually DATABASE_URL=jdbc:postgresql://${DATABASE_HOST}:${DATABASE_PORT/${DATABASE_DB} (do not include the username or password here)
 DATABASE_URL=jdbc:postgresql://db:5432/airbyte
+JOBS_DATABASE_MINIMUM_FLYWAY_MIGRATION_VERSION=0.29.15.001
```

Comment:
can you help me understand how these values would be updated in cloud?

Reply:
We would have to specify these values separately in our Helm manifests when we upgrade to a newer version of OSS. So I'll have to do some work in the manifest when we update Cloud to the version of Airbyte that contains the bootloader.

Does that answer your question?